finagle-core/src/main/scala/com/twitter/finagle/util/LoadService.scala

```diff
@@ -177,7 +177,7 @@ object LoadService {
 
     val classNamesFromRessources = for {
       rsc <- loader.getResources("META-INF/services/" + ifaceName).asScala
-      line <- ClassPath.readLines(Source.fromURL(rsc))
+      line <- ClassPath.readLines(Source.fromURL(rsc, "UTF-8"))
```

Comment:
apologies. didn't realize the Source api doesn't expose that.

i'll try to get this merged in soon.